datasetinsights/datasets/transformers/lspet2coco.py

```diff
@@ -0,0 +1,155 @@
+import json
+import shutil
+from pathlib import Path
+
+import numpy as np
+from coco import coco_keypoints, coco_skeleton
+from PIL import Image
+from scipy import io
+from tqdm import tqdm
+
+joints_lspet = [
+    "right_ankle",
+    "right_knee",
+    "right_hip",
+    "left_hip",
+    "left_knee",
+    "left_ankle",
+    "right_wrist",
+    "right_elbow",
+    "right_shoulder",
+    "left_shoulder",
+    "left_elbow",
+    "left_wrist",
+    "neck",
+    "head_top",
+]
+
+
+class LSPETtoCOCOTransformer:
+    def __init__(self, data_root):
+        self._data_root = Path(data_root)
+        annotation_file_path = self._data_root / "joints.mat"
```

Comment:
Let's make this a separate variable so that we don'e assume folder structure for images and labels.